HW8/optional_task_2.py

```diff
@@ -0,0 +1,26 @@
+# 1 option
+def check(check_lst):
+    if type(check_lst) is int:
```

Comment:
nit: nice funciton to check type is `isinstance()` and only condition can be return 
```suggestion
    return type(check_lst) is int
```